calendar_data/data.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+import requests
+from icalendar import Calendar
+
+
+def get_calendar_data(pyvo_events_url):
+    calendar_raw_data = requests.get(pyvo_events_url)
+    return calendar_raw_data.content
+
+
+def parse_calendar_data(pyvo_events_url):
```

Comment:
It took me a while to understand this function - it wasn't clear to me from the name that it only returns future events, might be better to name it with what its purpose is, as opposed to what it is doing. Maybe `get_future_events`?